src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -1,40 +1,20 @@
-import React, { useState, useReducer, useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
We are not using this right now, Am I correct? Please add an inline comment showing that this module will change.  @AuraAlba

Reply:
done